source/html/visualizer/src/app.ts

```diff
@@ -43,12 +43,9 @@ function renderCard(): HTMLElement {
 
     var adaptiveCard = new Adaptive.AdaptiveCard();
 
-    try {
-        adaptiveCard.hostConfig = Adaptive.parseHostConfig(currentConfigPayload);
-    }
-    catch (e) {
-        // TODO
-    }
+    
+    adaptiveCard.hostConfig = Adaptive.parseHostConfig(currentConfigPayload);
```

Comment:
Why remove the try...catch? If not caught, this exception will break the visualizer. It is swallowed by design.

Reply:
the exception will be caught by outer try..catch of tryRenderCard